### ISSUE TITLE
Print the SHA1 hash of HEAD somewhere

### ISSUE THREAD

**Issue by asmeurer** [sympy/sympy-bot-old#16]
If someone force pushes an update to a pull request, or even pushes a new commit, the test results are no longer valid.  So the SHA1 hash of the HEAD should be included in the report somewhere so that you can verify that the tests refer to the latest version of the branch.

**Comment by asmeurer** [comment]
Although it isn't so bad, because if you force push or add a new commit, it shows up below the comment with the test results.  But still, this would be nice to have.

**Comment by certik** [comment]
Yes, and also the sympy-bot should go over _all_ pull requests automatically and determine which ones are not uptodate. This requires to have a database of the HEAD sha1s in some google app engine app, and it can serve the html reports as well.

**Comment by asmeurer** [comment]
The name of the branch, number of the pull request, author, and SHA1 should all be at the top of the report.

Maybe also the Python version, architecture, and ground types should be prominent as well.

**Comment by certik** [comment]
And the hash of the master --- the tests submitted by sympy-bot should be colored: green (passes latest commit in the branch+master), yellow (passes the latest branch + older master), brown (passes the old branch+old master), red (fails).

**Comment by asmeurer** [comment]
I would use light green instead of brown if the only thing that's changed is master, since this very rarely causes test failures when there weren't any before (though it does often cause merge conflicts, but this would be a different category).

**Comment by certik** [comment]
I think this is implemented, so I am closing this one.

**Comment by asmeurer** [comment]
This is included in the summary, but not the report itself.  Actually, very little information is in the report itself.

And we never did the colors.
